src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -1624,6 +1624,29 @@ struct COMMAND_RPC_CREATE_WALLET
 	};
 };
 
+struct COMMAND_RPC_RESTORE_WALLET
+{
+	struct request
+	{
+		std::string filename;
+		std::string password;
```

Comment:
@fireice-uk A note: we transfer here a plain test password, thats not good.

Reply:
We give the user a choice to supply it from command line - this is similar to how wallet creation works